### PR TITLE
✨ manta small indels inform strelka calling

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

The main changes in this PR focus on collecting small indels from Manta and using those to inform Strelka. To accomplish this change, I've done the following:
1. Manta tool moves the small indel file and index to the working directory, renaming them in the process, then I collect those files as a single output (vcf file with secondary index)
1. Strelka tool now has an optional file input for the small indels as well as a toggle for using that file. If the file is provided and the user says to use the file, it will be used. In all other cases, the file will not be used.
1. Manta subwf and prod wf now collect the new small indel output from the Manta tool and returns it
1. Strelka subwf and prod wf now have an optional Manta small indel input and toggle for that file that it can hand to the tool
1. Combined workflow collects output from Manta and hands it to Strelka

Additional changes:
1. Missing bracket in Lancet prod wf fixed

Closes https://github.com/d3b-center/bixu-tracker/issues/667

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- [x] Strelka without Manta Indel https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/dccf7534-aabb-4547-a59c-8c9797313aad/#
- [x] Strelka with Manta Indel and no explicit toggle (defaults to not use it): https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/1d599ef0-1fda-4a34-8fc9-89b30a501e72/#
- [x] Strelka with Manta Indel and explicit off toggle: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/e0f412ca-09f8-4be7-9127-e242c6f8d172/#
- [x] Strelka with Manta Indel and on toggle: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/30bac167-7da8-4727-a026-6d08da51cd11/#
- [x] Manta https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/15d24419-33a5-47e0-b56b-ede1d0539608/
- [x] All changed tools pass cwltool validation

**Test Configuration**:
* Environment: cwltool 3.0.20200324120055, Cavatica
* Test files: see Cavatica test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: workflow/kfdrc_production_somatic_variant_cnv_wf.cwl [hunk 1]
```diff
@@ -475,6 +475,7 @@ steps:
       input_tumor_name: input_tumor_name
       input_normal_aligned: input_normal_aligned
       input_normal_name: input_normal_name
+      manta_small_indels: run_manta/manta_small_indels
```
Comment (migbro) [inline workflow/kfdrc_production_somatic_variant_cnv_wf.cwl:478]: hmmm, might consider making this optional as we haven't actually committed to making this our standard SOP. Need to run by @yuankunzhu  first...

Comment (yuankunzhu) [reply workflow/kfdrc_production_somatic_variant_cnv_wf.cwl:478]: let's make this optional.

**Comment by dmiller15** [comment]
New tests for optional flag for strelka incorporating manta small indels.

Without Manta Small Indels:
https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/bcad40ec-fe06-4956-9f84-7ea1cf88c693/

With Manta Small indels:
No value: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/451d334e-a5e2-4490-a559-f0b0e8881293/#
True: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/c9a3e1ee-3fb5-41af-883b-1d595fbeba9a/
False: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/df27ba70-beed-4ebe-800b-351d3890ff63/

File: workflow/kfdrc_production_somatic_variant_cnv_wf.cwl (hunk 1 above)
Comment (dmiller15) [reply workflow/kfdrc_production_somatic_variant_cnv_wf.cwl:479]: Ok I added a new variable that allows the value to be toggled. Had to rewrite some other stuff to fit it in. New tests for strelka below. Let me know if I missed any tests.

**Pull request merged**